aikau/src/main/resources/alfresco/forms/controls/DojoRadioButtons.js

```diff
@@ -23,6 +23,4 @@
  * @author Dave Draper
  * @deprecated Since 1.0.3 - Use [alfresco/forms/controls/RadioButtons]{@link module:alfresco/forms/controls/RadioButtons} instead
  */
-define(["dojo/_base/declare",
-        "alfresco/forms/controls/RadioButtons"], 
-        function(declare, RadioButtons) {});
\ No newline at end of file
+define([], function() {});
```

Comment:
Ah, I think I've actually made a mess of this deprecation. Compare with the changes I made to DojoSelect for example... DojoRadioButtons should be inheriting from RadioButtons now - could you correct this to actually do so please.